### ISSUE TITLE
Clear all selected when submit error, but not clear when submit success???

### ISSUE THREAD

**Issue by thearabbit** [aldeed/meteor-autoform-select2#37]
I have the problem when I submit form error, it clear all of selected value.
But when the submit form success, it not clear selected label (value is empty).

**Comment by KristerV** [comment]
yep, I also have this issue. @thearabbit did you ever get around this?

**Comment by thearabbit** [comment]
Oh, How the solution?

**Comment by KristerV** [comment]
I got around by using [selectize](https://atmospherejs.com/comerc/autoform-selectize). Thought it has exactly the same issue, but at least I could reset it "manually".

	AutoForm.addHooks(['insertProductForm'], {
		onSuccess: function(formType, result) {
			$('select')[0].selectize.clear()
		}
	})

Select2 also [should have this functionality](https://select2.github.io/options.html), but I just couldn't get `$.fn.select2.defaults.reset()` to work.

**Comment by thearabbit** [comment]
I tried `selectize`, but have any problem withe `onchage` order. for example I have 2 `selectize` and when I on change the first to change value of the second, it not work on the second.

**Comment by drew-y** [comment]
This worked for me, although it clears all selects.
```javascript
$('select').each(function() { //Select2 doesnt clear on its own
  $(this).select2('val', '');
});
```

**Comment by drew-y** [comment]
This worked for me, although it clears all selects.
```javascript
$('select').each(function() { //Select2 doesnt clear on its own
  $(this).select2('val', '');
});
```

**Comment by thearabbit** [comment]
Look great, it work fine on success hook.
But I still have problem with `onError` (all `select2` is clear).

**Comment by drew-y** [comment]
I Haven't had a chance to test the onError case yet.
